diff --git a/core/main.mk b/core/main.mk
index 6544742..58bb7aa 100755
--- a/core/main.mk
+++ b/core/main.mk
@@ -508,7 +508,7 @@ ifneq ($(dont_bother),true)
 subdir_makefiles := \
        $(shell build/tools/findleaves.py --prune=$(OUT_DIR) --prune=.repo --prune=.git $(subdirs) Android.mk)

-$(foreach mk, $(subdir_makefiles), $(eval include $(mk)))
+$(foreach mk, $(subdir_makefiles), $(info including $(mk)...)$(eval include $(mk)))

 endif # dont_bother



